apps/app/src/app/pages/pages/dashboard/dashboard.component.ts

```diff
@@ -51,20 +50,16 @@ export class DashboardComponent implements OnInit, OnDestroy {
       this.languageService.instant('pages.dashboard.title')
     );
 
-    this.routeDataSubscription = this.route.data.subscribe((data) => {
+    const subscription = this.route.data.subscribe((data) => {
```

Comment:
Tu peux directement écrire : 
` this.subscription.add(this.route.data.subscribe((data) => {
      if (data['applications']) {
        this.applications = data['applications'];
      }
    }));`

Reply:
Yes !